test-requirements.txt

```diff
@@ -11,3 +11,4 @@ boto3>=1.17.46
 composeml>=0.3.0
 urllib3>=1.26.5
 pyarrow>=3.0.0
+mock==4.0.3
```

Comment:
Can we use the builtin version of mock instead?

https://docs.python.org/dev/library/unittest.mock.html#unittest.mock.patch

Reply:
yep!